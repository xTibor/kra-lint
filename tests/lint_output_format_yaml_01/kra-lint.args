-C .kra-lint
-F yaml
