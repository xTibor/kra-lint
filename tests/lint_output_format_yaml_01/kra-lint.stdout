- path: sub_pass_01_01.kra
  messages:
  - message_title: Incorrect document name
    message_metadata:
    - metadata_type: expected
      metadata_content: not(contains("Test"))
    - metadata_type: found
      metadata_content: Test
- path: sub_pass_01_02.kra
  messages:
  - message_title: Incorrect document size
    message_metadata:
    - metadata_type: expected
      metadata_content: 256×256px/(any)dpi
    - metadata_type: found
      metadata_content: 128×128px/300dpi
